modules/loader/src/main/resources/application.conf

```diff
@@ -0,0 +1,4 @@
+snowplow {
```

Comment:
Can we use the same format as in main config? Or other way:

```
snowplow.input.retrievalMode = "FanOut"
snowplow.input.disableCloudWatch = false
```

Just the format of:

```
root {
  key = value
}
```

Feels very unusual.